### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by dontlaugh**
- Remove redundant README info
- Remove submodules update from scripts/bootstrap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,19 @@ The greymatter Operator enables a bootstrapped mesh deployment with automatic Gi
...
+
+```
+git clone git@github.com:greymatter-io/operator.git --recurse-submodules
+```
+
+If you already cloned the repository, initialize the submodules
+
+```
+git submodule update --init --recursive
+```
```
Comment (tilleryd) [inline README.md:25]: Mr too damn terse

File: README.md [hunk 2]
```diff
@@ -229,32 +235,3 @@ To launch, simply run:
...
-you'll also need the [OpenShift
-CLI](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/).
-
-## Setup
-
-Verify all dependency installations and update CUE modules:
-
-```bash
-./scripts/bootstrap
-```
```
Comment (tilleryd) [inline README.md:256]: What do you have against this block?

Comment (dontlaugh) [reply README.md:256]: It's mentioned above in the getting started section

Comment (tilleryd) [reply README.md:256]: Ah cool, I didn't look at the entire file.

**Review by tilleryd — APPROVED** [review]

**Pull request merged**